service/eveapi.py

```diff
@@ -346,19 +346,6 @@ def __getattr__(self, this):
         # perform arcane attribute majick trick
         return _Context(self._root, self._path + "/" + this, self.parameters)
 
-    def __call__(self, **kw):
-        if kw:
-            # specified keywords override contextual ones
-            for k, v in self.parameters.iteritems():
-                if k not in kw:
-                    kw[k] = v
-        else:
-            # no keywords provided, just update with contextual ones.
-            kw.update(self.parameters)
-
-        # now let the root context handle it further
-        return self._root(self._path, **kw)
-
```

Comment:
Yeah, looking it over again I don't think we actually should.  It's a little weird the way they're making it callable, but I reverted it.

This is one reason why I don't like including third party modules as direct parts of Pyfa, it exposes all the things they do that are weird/wrong. :)